docs/modules/plugins/pages/plugin-mobile-app.adoc

```diff
@@ -780,3 +780,60 @@ When I stop recording `Time Profiler` metrics and save results to file `${output
 include::partial$generic-ui-steps.adoc[]
 
 include::partial$proxy-steps.adoc[]
+
+=== Change network connection
+
+Turn ON/OFF network connection.
+
+[source,gherkin]
+----
+When I turn $toggle `$mode` network connection
+----
+* `$toggle` - The modes of the toggle: `ON` or `OFF`.
+* `$mode` - The name of the network connect to toggle.
+
+[cols="2,2,2", options="header"]
+|===
+
+|Mode
+|Platform
+|Description
+
+|`WIFI`
+|Android, iOS and emulators
+|Turn ON/OFF Wi-Fi connection.
+
+|`MOBILE_DATA`
+|Android, iOS and emulators
+|Turn ON/OFF Mobile Data connection.
+
+|`AIRPLANE`
+|Android and emulators
+|Turn ON/OFF Airplane Mode.
+
+|`WIFI_AND_MOBILE_DATA`
+|Android and emulators
+|Turn ON/OFF Mobile Data and Wi-Fi connections.
+
+|===
+
+TIP: For IOS devices: to return to the application you must use the step:
```

Comment:
is it for iOS devices only? are iOS simulators in scope?

Reply:
only for devices. Wi-Fi, mobile data, airplane mode aren't presented in simulators settings